.github/workflows/test.yml

```diff
@@ -1,26 +1,56 @@
 name: Tests
 on: [push, pull_request]
 jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-python@v5
+        with:
+          python-version: '3.10'
+      - name: Install requirements
+        run: pip install flake8 pycodestyle black[python2]==21.12b0 click==8.0.4 isort
+      - name: Check syntax and complexity
+        run: |
+          flake8 . --count --select=C901,E901,E999,F401,F821,F822,F823 --show-source --statistics
+      - name: Check codestyle
+        run: |
+          isort --diff --check ckanext/
+          black --diff --check ckanext/
+      - name: Check node location
+        run: |
+          echo "PATH=$PATH"
+          which node || echo "node not found"
+          node --version || echo "node failed"
+
   test:
+    needs: lint
+    strategy:
+      matrix:
+        include:
+          - ckan-version: "2.11"
+            ckan-image: "ckan/ckan-dev:2.11-py3.10"
+          - ckan-version: "2.10"
```

Comment:
do we need this?

Reply:
I would keep it, as this will make it visible whether the plugin is still compatible with older versions (considering others might want to use it, too?) If we ever face incompatibilities we can still see how to move on from there. WDYT?